elektronLogging/logger.go

```diff
@@ -0,0 +1,49 @@
+package elektronLogging
+
+import (
+	log "github.com/sirupsen/logrus"
+	. "github.com/spdfg/elektron/elektronLogging/types"
+	"strings"
+	"time"
+)
+
+var config LoggerConfig
+var logger *log.Logger
+var formatter ElektronFormatter
+var ElektronLog *LoggerImpl
+
+func BuildLogger(prefix string) {
+
+	// read configuration from yaml
+	config.GetConfig()
+
+	// create the log directory
+	startTime := time.Now()
+	formatter.TimestampFormat = "2006-01-02 15:04:05"
+	formattedStartTime := startTime.Format("20060102150405")
+	GetLogDir(startTime, "_")
```

Comment:
The second argument to GetLogDir() is a prefix. How is "_" a prefix? This is probably why the log directory name is not ending up prefixed with the one specified with the `-logPrefix` flag.

Reply:
Done